### ISSUE TITLE
Missing upickle ReadWriter for CheckDirection

### ISSUE THREAD

**Issue by lefou** [lolgab/mill-mima#56]
After overriding `mimaCheckDirection` in `build.sc`

```scala
  override def mimaCheckDirection: Target[CheckDirection] = T { CheckDirection.Backward }
```

I get the following error message:

```
Compiling /home/lefou/work/opensource/mill/build.sc
/home/lefou/work/opensource/mill/build.sc:190: could not find implicit value for parameter rw: upickle.default.ReadWriter[com.github.lolgab.mill.mima.CheckDirection.Backward.type]
  override def mimaCheckDirection: Target[CheckDirection] = T { CheckDirection.Backward }
                                                              ^
Compilation Failed
```

This is with Mill 0.10.5 and mill-mima 0.0.10 and 0.0.11.

**Comment by lefou** [comment]
Manually adding the following upickle `ReadWriter`s, fixes it locally:

```scala
  implicit val checkDirectionBackwardUpickleRW: ReadWriter[CheckDirection.Backward.type] = macroRW
  implicit val checkDirectionBothUpickleRW: ReadWriter[CheckDirection.Both.type] = macroRW
  implicit val checkDirectionForwardUpickleRW: ReadWriter[CheckDirection.Forward.type] = macroRW
  implicit val checkDirectionUpickleRW: ReadWriter[CheckDirection] = ReadWriter.merge(
    checkDirectionBackwardUpickleRW, checkDirectionBothUpickleRW, checkDirectionForwardUpickleRW
  )
```

**Comment by lolgab** [comment]
Thank you!
Fixed in #57
